### PR TITLE
Added ability to view Templates

### PR DESCRIPTION
**Pull request by chalettu**
PV - https://www.pivotaltracker.com/story/show/140582923.

Added ability to view Templates

### CODE REVIEW

**Comment by chalettu** [comment]
@miq-bot add_label euwe/no
@miq-bot add_label enhancement
@miq-bot add_label templates

**Comment by chalettu** [comment]
view template

File: client/app/templates/template-editor.html [hunk 1]
```diff
@@ -1,12 +1,45 @@
...
+        <li>
+          <a ui-sref="templates.explorer">{{'Templates'|translate}}</a>
+        </li>
+        <li class="active">{{vm.title}}</li>
+      </ol>
+  </span>
+</div>
+<div pf-toolbar class="section-toolbar section-toolbar-right-actions" config="vm.headerConfig" ng-if="vm.pageAction==='View'">
+  <actions>
+    <button class="btn btn-default primary-action" type="button" ng-click="vm.downloadTemplate()">
```
Comment (AllenBW) [inline client/app/templates/template-editor.html]: Need to translate this

File: client/app/templates/template-editor.html [hunk 2]
```diff
@@ -23,29 +56,33 @@ <h3 translate>Basic Information</h2>
             <div class="col-lg-7 col-md-7 col-sm-7 col-xs-7">
               <div class="form-horizontal">
                 <div class="form-group">
-                  <label class="control-label col-sm-4" translate>Name *</label>
+                  <label class="control-label col-sm-4" translate>{{'Name' | translate}} *</label>
```
Comment (AllenBW) [inline client/app/templates/template-editor.html]: Translate attached to the element not necessary here

**Comment by chriskacerguis** [comment]
@chalettu looks like during the AWS outage today something got really borked, can you update this PR so travis will rerun (I cannot restart it).

**Pull request merged**